Formula/wpscan.rb

```diff
@@ -0,0 +1,56 @@
+class Wpscan < Formula
+  desc "Black box WordPress vulnerability scanner"
+  homepage "https://wpscan.org"
+  url "https://github.com/wpscanteam/wpscan/archive/2.9.2.tar.gz"
+  sha256 "01c888bf2310354823a9522cd929e0caad5570c14eef6f47754c90e268fccad8"
+  head "https://github.com/wpscanteam/wpscan.git"
+
+  depends_on :ruby => "2.1.9"
+
+  def install
+    inreplace "lib/common/common_helper.rb" do |s|
+      s.gsub! "ROOT_DIR, 'cache'", "'#{var}/cache/wpscan'"
+      s.gsub! "ROOT_DIR, 'log.txt'", "'#{var}/log/wpscan/log.txt'"
+    end
+
+    if build.stable?
+      # Remove when a version >2.9.2 is released.
+      inreplace "lib/environment.rb", "require 'xmlrpc/client'", ""
+    end
+
+    system "unzip", "-o", "data.zip"
+    libexec.install "data", "lib", "spec", "Gemfile", "wpscan.rb"
+    if build.head?
+      # Merge into stable when a version >2.9.2 is released.
```

Comment:
I put `Gemfile.lock` in a patch along with the `xmlrpc/client` fix for Ruby 2.4+. The patch is from a random gist for now. I'll submit it to formula-patches if this formula is to be accepted and there won't be a release soon.